content/en/docs/rules/overriding.md

```diff
@@ -0,0 +1,273 @@
+---
+title: Overriding Rules
+description: Overriding Falco rules
+linktitle: Overriding Rules
+weight: 50
```

Comment:
I would suggest adding an alias to the old `appending` page so links won't be broken